src/README.md

```diff
@@ -1,31 +1,65 @@
-# OpenThread on EFR32MG12 Example
+# OpenThread on EFR32
+
+##### Table of Contents
+
+- [Prerequisites](#prerequisites)
+  - [Toolchain](#toolchain)
+  - [Flashing and debugging tools](#flash-debug)
+- [Building examples](#build)
+- [Flashing binaries](#flash)
+  - [Simplicity Commander](#simplicity-commander)
+  - [J-Link GDB Server](#jlink-gdb-server)
+  - [J-Link Commander](#jlink-commander)
+- [Run the example with EFR32 boards](#example)
+- [Additional features](#additional-features)
+- [Verification](#verification)
+
+---
 
 This directory contains example platform drivers for the [Silicon Labs EFR32MG12][efr32mg] based on [EFR32™ Mighty Gecko Wireless Starter Kit][slwstk6000b] or [Thunderboard™ Sense 2 Sensor-to-Cloud Advanced IoT Development Kit][sltb004a].
 
-[efr32mg]: http://www.silabs.com/products/wireless/mesh-networking/efr32mg-mighty-gecko-zigbee-thread-soc
+[efr32mg]: https://www.silabs.com/support/getting-started/mesh-networking/thread/mighty-gecko
 [slwstk6000b]: http://www.silabs.com/products/development-tools/wireless/mesh-networking/mighty-gecko-starter-kit
 [sltb004a]: https://www.silabs.com/products/development-tools/thunderboard/thunderboard-sense-two-kit
 
-The example platform drivers are intended to present the minimal code necessary to support OpenThread. [EFR32MG12P SoC][efr32mg12p] has rich memory and peripheral resources which can support all OpenThread capabilities. See the "Run the example with EFR32MG12 boards" section below for an example using basic OpenThread capabilities.
+The example platform drivers are intended to present the minimal code necessary to support OpenThread. [EFR32MG][efr32mg] has rich memory and peripheral resources which can support all OpenThread capabilities.
 
-See [sleepy-demo/README.md](sleepy-demo/README.md) for instructions for an example that uses the low-energy modes of the EFR32MG12 when running as a Sleepy End Device.
+See [EFR32 Sleepy Demo Example](./sleepy-demo/README.md) for instructions for an example that uses the low-energy modes of the EFR32MG12 when running as a Sleepy End Device.
```

Comment:
Here's another reference to EFR32MG12 that needs to be updated and the 12 needs to be removed.